templates/landscape/features.html

```diff
@@ -1,191 +1,243 @@
+                        alt="",
+                        width="568",
+                        height="853",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image objPosTop"}) | safe
+            }}
+          </div>
+        </div>
+        <h3 class="p-equal-height-row__item p-heading--5">Patch compliance - keep systems secure and up to date</h3>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted is-cover">
+            {{ image(url="https://assets.ubuntu.com/v1/d6b6ae6b-rbac.png",
+                        alt="",
+                        width="568",
+                        height="853",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <h3 class="p-equal-height-row__item p-heading--5">Role Based Access Control (RBAC)</h3>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted is-cover">
+            {{ image(url="https://assets.ubuntu.com/v1/bb8fd9a7-automated.png",
+                        alt="",
+                        width="568",
+                        height="853",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <h3 class="p-equal-height-row__item p-heading--5">Automated audit logging and compliance reporting</h3>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted is-cover">
+            {{ image(url="https://assets.ubuntu.com/v1/99d5b20b-regulatory-compliance.png",
+                        alt="",
+                        width="568",
+                        height="853",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image objPosTop"}) | safe
+            }}
+          </div>
+        </div>
+        <h3 class="p-equal-height-row__item p-heading--5">
+          Regulatory compliance is significantly simplified with custom reporting
+        </h3>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
+
+  <section class="p-section--shallow">
+    {%- call(slot) vf_tiered_list(is_list_full_width_on_tablet=false) -%}
```

Comment:
I notice that `vf_tiered_list` automatically applies shallow section on the list but on Design it does not have bottom padding and is wrapped in a regular section. Might be worth upstreaming to Vanilla to ask if we could have an option to remove bottom shallow section.

Reply:
Hmm, that could be something subjective. Our design didn't have a shallow section underneath, but some other design might have. It might still be a good idea to have a toggle option for that in the pattern itself, but we could also use `u-sv*` classes for these kind of usecases.